README.md

```diff
@@ -16,6 +16,10 @@ This use case shows how you can set up a Qlik Core application in a Google Kuber
 
 * Accept the EULA by modifying the `./qlik-core/engine-deployment.yaml` file.
 
+* Change the max number of sessions on an engine to 20 from 500 by changing `SESSIONS_PER_ENGINE_THRESHOLD` in the `./qlik-core/qix-session-service.yaml` file.
```

Comment:
IMO it sounds more natural to express changing "... from ... to ..." than "... to ... from ..."